cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtilsTest.java

```diff
@@ -0,0 +1,135 @@
+  private static final String REPLICATION_THROTTLE_STRING = "1000";
+  private static final String DEFAULT_REPLICATION_THROTTLE_STRING = "2000";
+  private static final String EXECUTION_PROGRESS_CHECK_INTERVAL_STRING = "1500";
+
+  @Test
+  public void testParseTimeRangeSet() {
+    HttpServletRequest mockRequest = EasyMock.mock(HttpServletRequest.class);
+    Map<String, String[]> paramMap = new HashMap<>();
+    paramMap.put(ParameterUtils.START_MS_PARAM, new String[]{ParameterUtils.START_MS_PARAM});
+    paramMap.put(ParameterUtils.END_MS_PARAM, new String[]{ParameterUtils.END_MS_PARAM});
+
+    EasyMock.expect(mockRequest.getParameterMap()).andReturn(paramMap).times(4);
+    EasyMock.expect(mockRequest.getParameter(ParameterUtils.START_MS_PARAM)).andReturn(START_TIME_STRING).times(2);
+    EasyMock.expect(mockRequest.getParameter(ParameterUtils.END_MS_PARAM)).andReturn(END_TIME_STRING).times(2);
+    EasyMock.replay(mockRequest);
+
+    Long startMs = ParameterUtils.startMsOrDefault(mockRequest, null);
+    Long endMs = ParameterUtils.endMsOrDefault(mockRequest, null);
+    Assert.assertNotNull(startMs);
+    Assert.assertNotNull(endMs);
+    Assert.assertEquals(Long.valueOf(START_TIME_STRING), startMs);
+    Assert.assertEquals(Long.valueOf(END_TIME_STRING), endMs);
+    ParameterUtils.validateTimeRange(startMs, endMs);
+
+    startMs = ParameterUtils.startMsOrDefault(mockRequest, DEFAULT_START_TIME_MS);
+    endMs = ParameterUtils.endMsOrDefault(mockRequest, DEFAULT_END_TIME_MS);
+
+    Assert.assertEquals(Long.valueOf(START_TIME_STRING), startMs);
+    Assert.assertEquals(Long.valueOf(END_TIME_STRING), endMs);
+  }
+
+  @Test
+  public void testParseTimeRangeNotSet() {
+    HttpServletRequest mockRequest = EasyMock.mock(HttpServletRequest.class);
+    // Mock the request so that it does not contain start/end time
+    EasyMock.expect(mockRequest.getParameterMap()).andReturn(Collections.emptyMap()).times(4);
+    EasyMock.replay(mockRequest);
+
+    Long startMs = ParameterUtils.startMsOrDefault(mockRequest, null);
+    Long endMs = ParameterUtils.endMsOrDefault(mockRequest, null);
+    Assert.assertNull(startMs);
+    Assert.assertNull(endMs);
+
+    startMs = ParameterUtils.startMsOrDefault(mockRequest, DEFAULT_START_TIME_MS);
+    endMs = ParameterUtils.endMsOrDefault(mockRequest, DEFAULT_END_TIME_MS);
+
+    Assert.assertEquals(DEFAULT_START_TIME_MS, startMs);
+    Assert.assertEquals(DEFAULT_END_TIME_MS, endMs);
+  }
+
+  @Test
+  public void testParseReplicationThrottleWithNoDefault() {
+    HttpServletRequest mockRequest = EasyMock.mock(HttpServletRequest.class);
+    KafkaCruiseControlConfig controlConfig = EasyMock.mock(KafkaCruiseControlConfig.class);
+
+    Map<String, String[]> paramMap = new HashMap<>();
+    paramMap.put(ParameterUtils.REPLICATION_THROTTLE_PARAM, new String[]{ParameterUtils.REPLICATION_THROTTLE_PARAM});
+
+    EasyMock.expect(mockRequest.getParameterMap()).andReturn(paramMap).times(1);
```

Comment:
(Applies to other similar uses in this test):
Nit: `times(1)` -> `once()` -- i.e. calling one time is special :)